lib/apiservers/portlayer/swagger.json

```diff
@@ -59,6 +59,115 @@
 				}
 			}
 		},
+		"/kv": {
```

Comment:
I originally had as path -- then ran into some self inflicted swagger wounds that were not related, but during the flailing I switched to query...happy to switch back as it's more inline with REST